server/package.json

```diff
@@ -62,10 +65,12 @@
     "bytes": "^3.1.2",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
+    "dataloader": "^2.2.2",
     "date-fns": "^2.30.0",
     "file-type": "16.5.4",
-    "graphql": "^16.8.0",
+    "graphql": "16.8.0",
     "graphql-fields": "^2.0.3",
+    "graphql-subscriptions": "2.0.0",
```

Comment:
This is needed by nestjs-query-graphql however I've disabled subscriptions for now